components/location/listItem.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from "react";
+
+const LocationListItem = ({
+  closePopups,
+  index,
+  location,
+  mapObject,
+  toggleLocationList,
+}) => {
+  const [onClick, setOnClick] = useState(null);
+
+  useEffect(() => {
+    if (location.marker && !onClick) {
+      // Reason why this arrow function looks so weird: https://stackoverflow.com/a/55621325
+      setOnClick(() => () => {
+        const popup = location.marker.getPopup();
+        if (!popup.isOpen()) {
+          // By including an index, we close every popup *except*
+          // for the one at the specified index.
+          closePopups(index);
+          toggleLocationList();
+          mapObject.jumpTo({
+            center: popup.getLngLat(),
+            essential: true,
+            zoom: 13,
+          });
+        } else closePopups();
+      });
+    }
+  }, [
+    closePopups,
+    index,
+    location.marker,
+    mapObject,
+    onClick,
+    toggleLocationList,
+  ]);
+
+  /** @todo
+   * - Populate card-badge-group class with content
+   */
+
+  return (
+    <div className="card location-list--item" onClick={onClick}>
+      <div className="card-content">
+        <div className="card-title">
+          {/* Kanad note: got rid of the weird `id` span and set the title to be the status label */}
+          <span
+            title={location.status.label}
+            className="card-status-indicator"
+            style={{ backgroundColor: location.status.accessibleColor }}
+          ></span>
+          <h2 className="name">{location.name}</h2>
+        </div>
+        {location.neighborhood && (
+          <h3 className="card-subtitle neighborhood">
+            {location.neighborhood}
+          </h3>
```

Comment:
Does urgent need also still need to make it in here?

Reply:
Yep! See these lines, which I have yet to migrate over:
https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/blob/8cb738911293f86aa7de13834a6da55e0551b033/components/location/listItem.jsx#L61-L65